pocs/camera/libfliconstants.py

```diff
@@ -0,0 +1,122 @@
+# c_long flidev_t
```

Comment:
My eyes glazed over at this and I'm just going to trust you. :)

Reply:
Not much more of a copy and paste from the header of the C library! Just had to change syntax from C to Python.